tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -111,46 +113,99 @@ type stressConfig struct {
 }
 
 // NewStresser creates stresser from a comma separated list of stresser types.
-func NewStresser(s string, sc *stressConfig, m *member) Stresser {
+func NewStresser(s string, sc *stressConfig, members []*member) Stresser {
 	types := strings.Split(s, ",")
 	if len(types) > 1 {
 		stressers := make([]Stresser, len(types))
 		for i, stype := range types {
-			stressers[i] = NewStresser(stype, sc, m)
+			stressers[i] = NewStresser(stype, sc, members)
 		}
 		return &compositeStresser{stressers}
 	}
+
+	endPts := make([]string, len(members))
+	for i, m := range members {
+		endPts[i] = m.ClientURL
+	}
+
+	rf := &runner.EtcdRunnerConfig{
```

Comment:
what is `rf`? there's no f in runner.EtcdRunnerConfig so what is it abbreviating?